ARCs/arc-0030.md

```diff
@@ -0,0 +1,71 @@
+---
+arc: 30
+title: Algorand standard for mutable collection metadata as an ASA.
+description: This is a standard for defining mutable, arbitrarily large collections of ASAs.
+author: Evan Maltz/Kinn Foundation @evan_maltz
+status: Draft
+type: Standards Track
+category: ARC
+created: 2022-08-12
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/25
+---
+
+# Standard for defining mutable collections of ASAs
+
+## Summary
+
+This document introduces the standard for declaring collection metadata, which represent an arbitrary group of indices.
+
+## Abstract
+
+This ARC introduces a standard for defining a collection ASA (cASA) linked to ARC19 metadata. This standard defines a collection as a mutable set of indices, providing flexibility. By linking to ARC19 metadata on IPFS, the set can be arbitrarily large, even including the complete set of all indices. Indices can be application IDs, ASA IDs, or addresses. 
+
+## Specification
+
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+> Comments like this are non-normative.
+
+To be considered a collection, the cASA **MUST** use the unit name `CLXN` and **MUST** contain a property `assets`. The `assets` property **MUST** contain an array with at least one index. A collection name and description are **RECOMMENDED**. All other fields are **OPTIONAL** and additional, custom fields **MAY** be included. The cASA **SHOULD** be minted using ARC19 to allow for immutability, but **MAY** be minted in the ARC3 format if immutability is preferred.
```

Comment:
It might be helpful to add `"url"` as **RECOMMENDED**. 
Maybe a `"social_media": {}` field can make sense to.  
What do you think?